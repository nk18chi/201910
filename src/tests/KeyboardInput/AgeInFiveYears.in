Percy_Bysshe_Shelley
34
