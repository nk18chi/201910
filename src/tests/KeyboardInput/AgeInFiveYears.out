Hello.  What is your name? 
Hi, Percy_Bysshe_Shelley!  How old are you? 
Did you know that in five years you will be 39 years old?
And five years ago you were 29! Imagine that!
